frontend/src/components/custom-tools/document-manager/DocumentManager.jsx

```diff
@@ -39,6 +40,22 @@ const viewTypes = {
   extract: "EXTRACT",
 };
 
+const base64toBlob = (data, mimeType) => {
+  const byteCharacters = atob(data?.data); // Decode base64
+  const byteArrays = [];
+
+  for (let offset = 0; offset < byteCharacters.length; offset += 512) {
+    const slice = byteCharacters.slice(offset, offset + 512);
+    const byteNumbers = new Array(slice.length);
+    for (let i = 0; i < slice.length; i++) {
+      byteNumbers[i] = slice.charCodeAt(i);
+    }
+    const byteArray = new Uint8Array(byteNumbers);
+    byteArrays.push(byteArray);
+  }
+  return new Blob(byteArrays, { type: mimeType });
+};
+
```

Comment:
This got updated because now we're supporting other doc types, so we need mimetype along with blob. Yes, I'll change the function name and move to GetStaticData helper.